### PR TITLE
Multi dc test

### PR DESCRIPTION
**Pull request by amoskong**
Add multiple datacenter tests

The test is based on multi-dc support.
- https://github.com/scylladb/scylla-cluster-tests/pull/301
- https://github.com/amoskong/scylla-cluster-tests/tree/multi-dc-dev

### CODE REVIEW

File: multiple_dc_test.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import logging
+
+from avocado import main
+
+from sdcm.tester import ClusterTester
+
+
+class MultipleDcTest(ClusterTester):
+    """
+
```
Comment (larisau) [inline multiple_dc_test.py]: it's worth to add the test description

File: multiple_dc_test.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                                              keyspace_num=1)
+        nodes_2nd_dc = [n for n in node_list if n.dc_idx == 1]
+        for node in nodes_2nd_dc:
+            self.stop_scylla(node)
+
+        for node in nodes_2nd_dc:
+            self.start_scylla(node)
+
+        results = self.get_stress_results(queue=stress_queue, stress_num=2, keyspace_num=1)
+
```
Comment (larisau) [inline multiple_dc_test.py]: no need to verify the stress results?

File: multiple_dc_test.py (hunk 1 above)
Comment (amoskong) [inline multiple_dc_test.py]: Ok

File: multiple_dc_test.py (hunk 2 above)
Comment (amoskong) [inline multiple_dc_test.py]: I just checked get_stress_results(), if some c-s process doesn't return result summary for some reason(error), get_stress_results() won't raise exception and just return a null list as subresult.

I will add an option for get_stress_results, to check valid result insider it. And we can also do some complex checking outside get_stress_results().

Thanks for the suggestion!

**Pull request merged**